### ISSUE TITLE
Unable to add users to the group

### ISSUE THREAD

**Issue by topraveenpaul** [shridhar-tl/jira-assistant#64]
Hi,
I was able to add myself in a user group, but unable to add other users( i have enough privileges in Jira) as it is not searching and pulling the users to the list. Basically nothing happens when i tried to add.

Is there any fix to this issue?

**Comment by jsouer** [comment]
I have the same problem. I assume it has everything to do with the new API due to privacy regulations. https://developer.atlassian.com/cloud/jira/platform/   ?

I'm waiting on this fix as well. Would be great to run the reports with more team members included.

**Comment by shridhar-tl** [comment]
@topraveenpaul & @jsouer - Sorry for delayed response. I understand the necessity of this feature. Soon I will have it implemented in the new version which is currently in BETA version.

**Comment by shridhar-tl** [comment]
Fixed this issue and will be available in the weekend release

**Comment by shridhar-tl** [comment]
Assuming this is working fine, closing this issue. If their is anything more to be done as part of this issue, this can be reopened or a new issue can be raised.
